source/scpp/extra/DUtils.h

```diff
@@ -74,3 +74,21 @@ VectorT duplicate(const VectorT& this_)
     VectorT dup = VectorT(this_);
     return dup;
 }
+
+template<typename T>
+int getCPPSizeof ()
+{
+    return sizeof(T);
+}
+
+template<typename T>
+void defaultCtorCPPObject (T* ptr)
+{
+    new(ptr) T();
+}
+
+template<typename T>
+void dtorCPPObject (T* ptr)
+{
+    ptr->~T();
+}
```

Comment:
Could https://github.com/dlang/druntime/blob/661c2d8975d5a5c6bfbab6b18850059aac857046/src/object.d#L3955-L3971 help ?

Reply:
I'll give it a try but in the cases where linker can't link `~object()` directly, that probably won't work either.